src/_restore.sh

```diff
@@ -77,7 +77,7 @@ function restore_doPostgresRestore() {
       debug "`compose_client exec -T $postgresComposeService\
         tar -xvzf /backups/$backupFile ./$db.psql 2>&1`"
       log "Restoring $db"
-      dbRestoreFlags="-d $db --clean --if-exists --no-owner --disable-triggers --verbose"
```

Comment:
@jesseops Do you recall if there was a specific reason that this `--no-owner` flag was being used?

Reply:
My goal here was to handle situations where the database user differed - eg, in some environment `core_admin` was `svc-plextrac-postgres-core-admin` or something similar. The idea being to use the _current_ db setup as the source of truth for roles/permissions. It's possible that this is an unnecessary precaution or one we can resolve through a runbook instead.

Reply:
I was initially thinking that this could be addressed through some kind of role mapping, eg

```
dbAdminRole=<do something to map `core` to $PG_CORE_ADMIN here>
dbRestoreFlags="-d $db --clean --if-exists --role=$dbAdminRole --no-owner --disable-triggers --verbose"
```

Reply:
I saw your suggestion about `--role=core_admin` in the Jira ticket and that was the first thing i tried. I couldn't get it to work and not sure exactly what i was doing wrong.

Reply:
Here's what I came up with (I ran into a snag where I was using `DB_<db>_ADMIN` instead of `PG_<db>_ADMIN_USER` :doh:)

![Screenshot from 2023-01-27 14-43-09](https://user-images.githubusercontent.com/4613566/215206324-2946bf6b-5616-44ff-978d-142f2f94d05c.png)
![Screenshot from 2023-01-27 14-42-53](https://user-images.githubusercontent.com/4613566/215206327-7d418dea-3b72-4c88-84e5-0f8de8a8b980.png)

Reply:
:embarrassed: ok maybe `--role` and `--no-owner` are incompatible.
![Screenshot from 2023-01-27 14-46-12](https://user-images.githubusercontent.com/4613566/215206595-42187369-682d-499c-94f0-83cfcca47db1.png)

Reply:
OH well if you do a restore with one user and THEN try to set role to another it doesn't happy. :badpokerface: trying a new test case where

1. Postgres is freshly setup
2. Backup taken as normal
3. Verify table owner is `core_admin`
4. Perform restore using `internalonly` user but drop role to `core_admin`
5. Validate table owner remains the same

Reply:
I _think_ that works

![image](https://user-images.githubusercontent.com/4613566/215211558-6183d658-cce9-4164-9e15-6daec3daa494.png)

Reply:
https://github.com/PlexTrac/plextrac-manager-util/pull/55/files happy to chat through this path and pros/cons

Reply:
UGH well turns out it's more complex than all that because the permissions are included in the backup :/ 
![image](https://user-images.githubusercontent.com/4613566/215213840-992252c0-849d-4965-a1fc-73ce44b78467.png)

Reply:
Ok, if I add `--no-privileges` the restore finishes and I see the correct table owner. Investigating if the privileges are correct all the way down due to the original permission grants or if this is an issue

![image](https://user-images.githubusercontent.com/4613566/215214604-7da955ef-b862-475b-87be-374034127271.png)

Reply:
I _think_ that does it 
![image](https://user-images.githubusercontent.com/4613566/215214823-88f565a4-9c3c-41d0-bf01-de9edf047a14.png)

Reply:
Nice! Pulling in your changes to test them now

Reply:
This was definitely the area I was most concerned about when putting together the initial restore stuff, so it's a) a bummer it did end up having issues and b) satisfying to _hopefully_ have the desired end result functioning, more or less 😂 


Let me know how it goes. If this is still an issue we can just move to the change you had already and have a runbook for changing the owner/grants if needed.

Reply:
Sweet, this worked perfectly. I was able to perform the `plextrac restore` and have the table ownership be correct. And i validated that by performing a schema migration ('down' and then back 'up') and the core_admin and runbooks_admin users were able to modify the schema without issue.

This looks ready to go from my testing. @jesseops Feel free to approve and merge once you are comfortable with it on your end.